### DISCUSSION TITLE
Hvordan sorteres opgaver og vagter under events?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#26] [Questions] [answered]
Hvordan sorteres opgaver og vagter under events lige nu? 

Der er noget ulogisk i den nuværende sortering. Se f.eks. Ungdommens Folkemøde: https://rehelper.dk/portal/kulturens/event/ungdommens-folkemode-2iy9

Det ville være mere logisk med en kronologisk sortering således, at opgaver sorteres efter tidligste vagtdag først. I tilfældet Ungdommens Folkemøde ville f.eks. opstilling komme først og oprydning komme sidst.

**Comment by mikkelsl** [comment, marked as answer]
Det sorteres efter enten efter senest oprettede/opdaterede.

Der er en opgave på det her, I kan prioritere i Volunteering: https://github.com/orgs/xooom-community/projects/2?pane=issue&itemId=61362341
